tasks/condition/condition.js

```diff
@@ -5,34 +5,80 @@
  */
 {
   const a = 3,
-        b = 2;
+    b = 2;
 
   if (a + b < 4) {
     console.log('wrong');
   } else {
     console.log('too much');
   }
 }
+//TaskDONE - rewrite IF..ELSE to ternary operator
+{
+  const a = 3,
+    b = 2;
 
+  a + b < 4 ? console.log('wrong') : console.log('too much');
+}
 /** TODO
  * The system receives two params x and y.
  * Implement the logic when z equal to sum of x and y in case both of them less than 10 or multiplication in case of greater or equal than 10
  * Task 1 - if else
  * Task 2 - ternary operator
  */
 {
-    const x = 3;
-    const y = 7;
+  const x = 3;
+  const y = 7;
+}
+// Task 1 - if else:
+{
+  const x = 3;
+  const y = 7;
+
+  let z;
+  if ((x < 10) && (y < 10)) {
```

Comment:
inner brackets are not necessary, 
if ( x < 10  &&  y < 10 )

Reply:
Done